brazil.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Since GluonCV doesn't have this file `brazil.code-workspace ` before, I was wondering do we need it here?